actors/builtin/multisig/multisig_actor.go

```diff
@@ -93,7 +94,7 @@ type ProposeParams struct {
 
 func (a Actor) Propose(rt vmr.Runtime, params *ProposeParams) *cbg.CborInt {
 	rt.ValidateImmediateCallerType(builtin.CallerTypesSignable...)
-	callerAddr := rt.Message().Caller()
+	callerAddr := resolveToPubKey(rt, rt.Message().Caller())
```

Comment:
Almost every method on the Multisig actor is going to need to make this call. Holding off on completing the fix here until I am told this is the right solution.